projects/cashmere/src/lib/sort/sort.spec.ts

```diff
@@ -42,7 +42,7 @@
 //
 //   let component: SimpleMatSortApp;
 //
-//   beforeEach(async(() => {
+//   beforeEach(waitForAsync(() => {
```

Comment:
do we really need this file?

Reply:
I think this entire file of unit tests is commented out.  Not sure the history of that, but I'm in favor of removing.  Will do, and same for table.